lib/publiccloud/utils.pm

```diff
@@ -18,9 +18,13 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils qw(is_sle is_public_cloud get_version_id);
+use version_utils qw(is_sle is_public_cloud get_version_id is_transactional);
+use transactional qw(check_reboot_changes trup_call process_reboot);
 use registration;
 
+# Indicating if the openQA port has been already allowed via SELinux policies
+my $openqa_port_allowed = 0;
```

Comment:
Will this variable really persist between let's say test modules?

Reply:
Yes, otherwise the VR would have failed. But I'll check again, to be sure

Reply:
It's not executed in https://duck-norris.qe.suse.de/tests/13480#step/enable_selinux/14, so I take this as a clear sign that it works.